packages/server/src/service/contact/normalize.ts

```diff
@@ -1,7 +1,7 @@
 import { RawContactRecord, ContactRecord, RawContact, OptionalLocale } from "./type"
 import { AvailableState } from "../../common"
 import { mandatoryTransform } from "./transformers"
-import { e164 } from '../twilio'
+import { e164 } from '../../common/util'
```

Comment:
In here, you need to run `contact.faxes.forEach(checkE164)`.  You can get rid of `normalizeContact`.